packages/legacy/core/App/navigators/MainStack.tsx

```diff
@@ -20,10 +20,14 @@ import TabStack from './TabStack'
 import { useDefaultStackOptions } from './defaultStackOptions'
 import CredentialDetails from '../screens/CredentialDetails'
 import OpenIDCredentialDetails from '../modules/openid/screens/OpenIDCredentialDetails'
+import { useTour } from '../contexts/tour/tour-context'
+
+import { View } from 'react-native'
```

Comment:
Package imports should be grouped above relative imports, preferably in alphabetical order